Makefile

```diff
@@ -128,12 +106,55 @@ run-kube-proxy:
 	-docker rm -f calico-kube-proxy
 	docker run --name calico-kube-proxy -d --net=host --privileged gcr.io/google_containers/hyperkube:v$(K8S_VERSION) /hyperkube proxy --master=http://127.0.0.1:8080 --v=2
 
-kubectl:
-	wget http://storage.googleapis.com/kubernetes-release/release/v$(K8S_VERSION)/bin/linux/amd64/kubectl
-	chmod 755 kubectl
-
 dist/calicoctl:
-	mkdir -p dist
-	sudo chmod a+w dist
-	curl -o dist/calicoctl -L https://github.com/projectcalico/calico-containers/releases/download/v0.17.0/calicoctl
+	curl -o dist/calicoctl -L https://github.com/projectcalico/calico-containers/releases/download/v$(CALICO_NODE_VERSION)/calicoctl
 	chmod +x dist/calicoctl
+
+.PHONY: vendor-up
+vendor-up:
```

Comment:
Could you please add some comments describing what this and the other new makefile targets do? Why / when do I want to run this instead of just running "glide up"